scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -63,13 +63,15 @@ object Common {
 
     FileInputFormat
       .getInputPaths(conf)
-      .map { path =>
-        val fs = path.getFileSystem(conf)
-        val size = fs.globStatus(path)
-          .map(status => fs.getContentSummary(status.getPath).getLength)
-          .sum
-
-        path -> size
+      .flatMap { path =>
+        Try {
+          val fs = path.getFileSystem(conf)
+          val size = fs.globStatus(path)
```

Comment:
That said, I'm not sure you want to be failing the job here if you have an exception. Presumably we want to just log and give warnings that reducer estimation may be off and go ahead assuming they were empty?